opencti-platform/opencti-graphql/src/migrations/1679582135178-separate-case-types.js

```diff
@@ -0,0 +1,54 @@
+import { Promise as BluePromise } from 'bluebird';
+import { findAll as getAllCases } from '../modules/case/case-domain';
+import { addVocabulary, findAll as getVocabularies } from '../modules/vocabulary/vocabulary-domain';
+import { executionContext, SYSTEM_USER } from '../utils/access';
+import { addCaseIncident } from '../modules/case/case-incident/case-incident-domain';
+import { addFeedback } from '../modules/case/feedback/feedback-domain';
+import { buildCompleteUser, findById } from '../domain/user';
+import { openVocabularies } from '../modules/vocabulary/vocabulary-utils';
+import { VocabularyCategory } from '../generated/graphql';
+
+export const up = async (next) => {
```

Comment:
Maybe I miss something
But, you don't remove case_types_ov and old Case entity right ?